docs/styles.md

```diff
@@ -0,0 +1,73 @@
+# Styling content for RAISE
+
+In order to maintain consistency and reliability across all RAISE content, the styling associated with RAISE content is consolidated into a series of css classes located in `src/styles`. 
+
+RAISE Style classes are either written for specific elements, or can be applied to a variety of objects. Those differences are listed below. Content Developers 
+
+
+## Gray Box (os-raise-graybox)
+Add a Gray box around your element
+
+**Example**
+<div style="text-align: center;">
+    <img src="./static/graybox.png" width="350">
```

Comment:
I would fix this screenshot per [this comment](https://github.com/openstax/k12/issues/274#issuecomment-1284114779) to not reflect the ability to style interactive blocks as that will break outside of Moodle storage + is not something we want content devs to try and do since the styling / rendering of interactive blocks should be self-contained in the React code.